docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,230 @@
+# Battery Monitor
+
+You can access the information related to the power consumption by applications or by hardware resources on a battery-powered device for a certain duration of time. The value is returned as the percentage of the total batttery capacity.
+
+The power consumption of an application is calculated by accumulating the usage of various individual resources used by the application.
+
+The Battery Monitor APIs provide following provisions to fetch the battery usage:
+
+- Getting battery usage information of a particular application.
+
+  You can [retrieve information of an application](#appusage_get) by specifying its application ID, [resource](#resource_key) IDs, and the time [duration](#duration_key).
+
+- Getting battery usage information of a particular resource.
+
+  You can [retrieve information of a single or muiltiple resources](#resourceusage_get) by specifying its [resource](#resource_key) IDs, and the time [duration](#duration_key).
+
+## Prerequisites
+
+To enable your application to use the Battery Monitor functionality:
+
+1. To use the functions and data types of the Battery Monitor API (in [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__BATTERY__MONITOR__MODULE.html) applications), include the `<battery_monitor.h>` header file in your application:
+
+   ```
+   #include <battery_monitor.h>
+   ```
+
+2. The application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+   ```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/systemmonitor</privilege>
+   </privileges>
+   ```
+
+<a name="appusage_get"></a>
+## Getting Battery Usage Information of Particular Application
+
+The battery usage information contains a combination of application ID, resource ID, and time duration.
+
+1. Fetching information of an application for a particular resource over a certain duration of time.
+   The `battery_monitor_resource_id_e`, `battery_monitor_duration_type_e` enums can be used to provide information related to specific resource and particular duration type respectively, as shown in the following code:
+
+   ```
+   #include <battery_monitor.h>
```

Comment:
Global. Please use 4 spaces instead of a tab for the indentation.

Reply:
@mijin85cho 
I would like your opinion on this one. 
I have used 3 spaces for the indentation (As the other guides followed the same). 
Do you want me to change it to 4 spaces instead?

Reply:
@d1-agarwal 
In Tizen docs, we use 4 space. So kindly change from 3 to 4 spaces.